question1/demo.js

```diff
@@ -0,0 +1,81 @@
+const given_array = [
+    {
+      "FirstName": "John",
+      "LastName": "Doe",
+      "Age": 20,
+      "Department": "Computer Science"
+    },
+    {
+      "FirstName": "Jane",
+      "LastName": "Smith",
+      "Age": 22,
+      "Department": "Physics"
+    },
+    {
+      "FirstName": "Michael",
+      "LastName": "Johnson",
+      "Age": 21, 
+      "Department": "Mathematics"
+    },
+    {
+      "FirstName": "Sarah",
+      "LastName": "Williams",
+      "Age": 19,
+      "Department": "Computer Science"
+    },
+    {
+      "FirstName": "Robert",
+      "LastName": "Brown",
+      "Age": 23,
+      "Department": "Mathematics"
+    },
+    {
+      "FirstName": "Emily",
+      "LastName": "Davis",
+      "Age": 20,
+      "Department": "Computer Science"
+    }
+  ]
+
+  
+  
+  
+  //   1.List the students whose department is computer science.
+  console.log(
+    given_array
+    .filter( x => x.Department == "Computer Science")
+    );
+
+
+    //   2. List the first name of students whose age is greater than  21
+
+  //console.log(given_array.filter(x => x.Age > 21).map(x => x.FirstName,x.Age))
+
+  console.log(
+    given_array
+      .filter(x => x.Age > 21)
+      .map(x => ({ FirstName: x.FirstName, Age: x.Age }))
```

Comment:
do we need age here? given_array
      .filter(x => x.Age > 21).map(x => x.FirstName) enough right?